prisma/migrations/20240704102523_init/migration.sql

```diff
@@ -1,6 +1,6 @@
 -- CreateTable
 CREATE TABLE "User" (
-    "id" SERIAL NOT NULL,
+    "id" TEXT NOT NULL,
     "email" TEXT NOT NULL,
```

Comment:
If you have to change existing database schema then I would create a new migration for it